apps/shipments/iot_client.py

```diff
@@ -36,3 +42,93 @@ def update_shadow(self, device_id, shadow):
             raise AWSIoTError("Error in response from AWS IoT")
 
         return iot_shadow['data']
+
+    def get_list_owner_devices(self, owner_id, next_token=None, results=None, **kwargs):
+        LOG.debug(f'Getting devices for: {owner_id} from AWS IoT')
+        log_metric('transmission.info', tags={'method': 'DeviceAWSIoTClient.get_list_owner_devices'})
+
+        active = kwargs.get('active', None)
+        in_bbox = kwargs.get('in_bbox', None)
+
+        if in_bbox:
+            in_bbox = ','.join([c.strip() for c in in_bbox.split(',')])
+            in_bbox_is_valid, message = self.validate_in_bbox(in_bbox)
+            if not in_bbox_is_valid:
+                raise ParseError(message)
+
+        if not next_token:
+            results = []
+
+        params_dict = OrderedDict(ownerId=owner_id,
```

Comment:
Is Owner_id required for this call? Because if it is, then this endpoint ought to be disabled if profiles is.

Reply:
Same as above

Reply:
@jamesfneyer owner_id should exist, even if profiles is disabled. The difference is that when profiles is disabled, the endpoint should accept an `owner_id` parameter, instead of pulling the users ID from the JWT.